cmd/apiserver/apiserver.go

```diff
@@ -27,8 +27,6 @@ import (
 	"k8s.io/kubernetes/pkg/util/logs"
 
 	"github.com/kubernetes-incubator/service-catalog/cmd/apiserver/app/server"
-	// install our API groups
-	_ "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/install"
```

Comment:
@simonleung8 I think this line needs to go back in.

- The controller manager and API server have to install the API group independently, since they are run in different processes
- API installation in tests does not affect production code

So, I think this will actually break the API server, and the our tests don't detect it at the moment because #424 isn't merged.

Please put this line back in; LGTM otherwise.